command-line/test/Command/Script/HashSpec.hs

```diff
@@ -44,85 +44,85 @@ spec = do
         specScriptHashProper "script1w8469gq5ed7xtyf2tqdng5yn7ykgckkfcl38xre8hk3ejk2lcwt"
             [iii|#{verKeyH4}|]
 
-        specScriptHashProper "drep_script13hu7a632tuntrlyjkazzczrzzt7dx6e8v3rc25jd9zcgv68nx9r"
+        specScriptHashProper "drep1ydv8xcdxwpy7zrf805yenpz7cdnap4c87v7zewvmdjyyygq44jy4j"
```

Comment:
Looking at those strings makes me wish those were in a text file usable externally and loaded by our tests so as to be relatively simple to edit and publish